components/NavBar.tsx

```diff
@@ -1,28 +1,45 @@
+"use client";
 import Image from "next/image";
 import Link from "next/link";
-import { Lobster } from "next/font/google";
+import { Lobster, Inter } from "next/font/google";
+import { useSession } from "next-auth/react";
 
 const lobster = Lobster({ weight: "400", subsets: ["latin"] });
-import Icon from "./Icon";
+const inter = Inter({ weight: "400", subsets: ["latin"] });
+import { SessionProvider } from "next-auth/react";
 
 export default function NavBar() {
+  const { data: session, status } = useSession();
   return (
-    <header
-      className={`flex justify-between items-center border-b border-gray-800 p-4 ${lobster.className}`}
-    >
-      <Link href="/">
-        <h1 className="text-2xl">Collections</h1>
-      </Link>
+    <SessionProvider>
```

Comment:
this should go in your root `Providers` component since you'll be using it everywhere